DMan/HW5/5.3.py

```diff
@@ -0,0 +1,15 @@
+# honestly found it on the internet:
+
+# import math
+# n = int(input("Enter a number to calculate its factorial"))
+# print(math.factorial(n))
+
+
+num = int(input("Enter a number to calculate its factorial"))
```

Comment:
good job
look at this implementation
input_number = int(input("Input natural number: "))

fact = 1

if input_number == 1 or input_number == 0:
    print("Factorial {}! =".format(input_number), fact)
elif input_number < 0:
    print("Factorial non positive number don't exist")
else:
    for item in range(1,input_number + 1):
        fact *= item
    print("Factorial {}! =".format(input_number), fact)

Reply:
Great! Thank you for this. I knew nothing about factorials. Now I will